cluster/gce/configure-vm.sh

```diff
@@ -661,7 +688,17 @@ if [[ -z "${is_push}" ]]; then
   remove-docker-artifacts
   run-salt
   set-good-motd
+
+  curl-metadata k8s-user-startup-script > ${INSTALL_DIR}/k8s-user-script.sh
+  user_script=$(cat ${INSTALL_DIR}/k8s-user-script.sh)
+  if [[ ! -z ${user_script} ]]; then
+    wait-for-cluster-healthy-or-bust
+    chmod u+x ${INSTALL_DIR}/k8s-user-script.sh
+    echo "== running user startup script =="
+    ${INSTALL_DIR}/k8s-user-script.sh
```

Comment:
Right now we aren't using a subshell (e.g. `out=$("${INSTALL_DIR}"/k8s-user-script.sh); echo $out`) and I was wondering if we should be.